packages/components/psammead-grid/src/index.jsx

```diff
@@ -335,8 +319,14 @@ Grid.propTypes = {
     group5: number.isRequired,
   }).isRequired,
   enableGelGutters: bool,
-  enableGelMargins: bool,
   enableNegativeGelMargins: bool,
+  margins: shape({
```

Comment:
We might want to simplify https://github.com/bbc/psammead/blob/margins-update/packages/components/psammead-grid/src/index.jsx#L323-L329 for scenarios where all margins are enabled for all breakpoints. Perhaps passing in an optional `enableAllMargins: true` which should set margins to true for all breakpoints

Reply:
I agree. Can we do this in a follow-up PR? For all of our Articles/FrontPages/current Simorgh pages, we will need to have margins on for groups 0-3 and off for 4-5, so we could simplify in that way, too.